frontend/packages/integration-tests-cypress/tests/storage/restore.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { testName, checkErrors } from '../../support';
+import { dropdownFirstItem, RestoreDetails } from '../../views/storage/snapshot';
+import { listPage } from '../../views/list-page';
+import {
+  PVC,
+  testerDeployment,
+  SnapshotClass,
+  Snapshot,
+  PatchForVolume,
+} from '../../mocks/snapshot';
+import { modal } from '../../views/modal';
+import { waitForResourceStatusToContain } from '../../views/common';
+
+// These tests are meant to be run on AWS as only AWS supports CSI storage classes(gp2-csi)
+if (Cypress.env('BRIDGE_AWS')) {
+  describe('Snapshot Tests', () => {
+    before(() => {
+      cy.login();
+      cy.createProject(testName);
+      cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc apply -f -`);
+      cy.clickNavLink(['Storage', 'Persistent Volume Claims']);
+      listPage.filter.byName(PVC.metadata.name);
+      waitForResourceStatusToContain('Bound');
+      cy.clickNavLink(['Volume Snapshots']);
+      cy.exec(`echo '${JSON.stringify(Snapshot)}' | oc apply -n ${testName} -f -`);
+      waitForResourceStatusToContain('Ready');
+    });
+
+    afterEach(() => {
+      checkErrors();
+    });
+
+    after(() => {
+      cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc delete -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(PVC)}' | oc delete -n ${testName} -f -`);
+      cy.exec(`oc delete pvc ${Snapshot.metadata.name}-restore -n ${testName}`)
+        .its('stdout')
+        .then(() => {
+          cy.exec(`echo '${JSON.stringify(Snapshot)}' | oc delete -n ${testName} -f -`);
+          cy.deleteProject(testName);
+          cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc delete -f -`);
+          cy.logout();
+        });
+    });
+
+    it('Restore a Snapshot to create a new claim from it', () => {
+      listPage.rows.clickKebabAction(Snapshot.metadata.name, 'Restore as new PVC');
+      modal.shouldBeOpened();
+      cy.get(RestoreDetails.restoreName).should('have.value', `${Snapshot.metadata.name}-restore`);
```

Comment:
```suggestion
      cy.byTestId('pvc-name').should('have.value', `${Snapshot.metadata.name}-restore`);
```
No need to use a constant since it's only used in one place.